webapi/Options/PromptsOptions.cs

```diff
@@ -161,4 +163,29 @@ public class PromptsOptions
     /// </summary>
     /// <returns>A shallow copy of the options.</returns>
     internal PromptsOptions Copy() => (PromptsOptions)this.MemberwiseClone();
+
+    /// <summary>
+    /// Tries to retrieve the memoryName asociated with the specified memory type.
+    /// </summary>
+    internal bool TryGetMemoryName(string memoryType, out string memoryName)
```

Comment:
You probably want to name this TryGetMemoryType() ?

Reply:
No actually this method is using the memoryType to get the memoryName. Think about it this way, we define two types of memory, LongTermMemory, and WorkingMemory. The actual names of the indexes or tags that the memories store uses for those memory types are configurable and therfore can be changed, so we need a way to map from the typo of memory asked for in the API, to the name of the index or tag where it's stored in the memory store..

Reply:
So, really, it's GetMemoryTypeName()?

Reply:
You're right, that would be more correct, although maybe more confusing at this point. I'll change it to TryGetMemoryContainerName